### DISCUSSION TITLE
Skin elements (such as UR bar) should be able to bypass hide interface feature (shift+tab)

### DISCUSSION THREAD

**Discussion by KaktuszSok** [ppy/osu#27874] [Ideas] [answered]
In stable when you play with the interface off (shift + tab), the UR bar still appears. In lazer the UR bar gets hidden as well, which some people may not like. Adding an option on UI elements to bypass UI hiding would allow users to address this discrepancy if they wish.

**Comment by Walavouchey** [comment, marked as answer]
this has already been suggested in https://github.com/ppy/osu/discussions/19486

please upvote that thread instead to show support for such a feature
